CLI Implementation 🚀

QuintonC opened the pull request.

QuintonC committed:
Fixing issues with branch.

drewradcliff commented on src/components/Row/index.tsx:
```diff
@@ -1,89 +1,96 @@
-  gray,
-  green,
-  yellow,
+	darkendGreen,
```
Check your formatter not using prettier format defaults

drewradcliff commented on src/types/index.tsx:
```diff
@@ -9,6 +9,12 @@ export interface Guess {
   matches: Match[];
 }
 
+export interface Matches {
```
These types were removed before because they aren't used

drewradcliff commented:
I will test this locally probably tomorrow

QuintonC committed:
Resolved PR issues. Updated color names as I found a small visual bug regarding colors and felt it was best to go ahead and resolve it.

QuintonC committed:
Fixed a couple of other color issues.

QuintonC committed:
Merge branch 'main' into cli-implementation

* main:
  Add MWResponse type
  Add dictionary api and env cariables

drewradcliff committed:
Stupid port stuff

drewradcliff committed:
Update README :fire:

QuintonC committed:
:memo: Updated README

Started on README template, will add more as we continue developing the project

drewradcliff reviewed (approved):
nice
